### ISSUE TITLE
Project status / roadmap 

### ISSUE THREAD

**Issue by itsezc** [AEB-labs/cruddl#140]
I have actively been following the development of Cruddl for a while now, I am eager to get involved in the development process, but there is a lot of missing information such as the project objectives, roadmap, indepth documentation (w/ Examples) and more.

Key features such as:

- Subscriptions
- Live Queries
- Seeding
- Custom resolver logic (like with Prisma)
- Apollo Federation support

Other factors to consider are:

- Documentation
- Examples
- Boilerplates
- Blog (with Updates / Progress?)

All these would not only boost the project's awareness but also use cases and contributions.

**Comment by itsezc** [comment]
@Yogu any feedback / input on this matter?

**Comment by ManAnRuck** [comment]
+1

**Comment by tomdavidson** [comment]
Ive been watching with excitement and curiosity as well. ArangoDB has a GraphQL Foxx Service but I did not know how it compares with cruddl.

**Comment by mfusser** [comment]
ch würd mal in den Cruddl thread zur roadmap folgendes schreiben:
Hello everyone,
sorry for the late answer.
We have been very short on time, so we could not properly take care of communication.
We are still very busy but we will try to improve our communication and at least try to answer your questions.
Regarding the roadmap:
We currently do not have any larger features planned.
We have ideas both about GraphQL subscriptions and using FoxxServices but we currently do not have plans to implement anything in the near future (although that might change if we run into an urgent use case)

@tomdavidson I am not sure what you mean by your question as to me Arango Foxx Services and Cruddl are two very different things (I am not an expert on Foxx Services though).

**Comment by mfusser** [comment]
@tomdavidson ah sorry i misunderstood. 
At first glance GraphQL Foxx Service seems like a similar concept although I think cruddl has a few additional features like relations, permissions and time-to-live options 
Though as I do not know GraphQL Foxx Service it is hard to say.

**Comment by tomdavidson** [comment]
Foxx is also in an synchronous runtime but has direct access to the data.  Seems like there are different opportunities with cruddl and the GraphQL Foxx Service but Im not exactly sure. A comparison in the docs could be handy to new cruddl users & contributors.

**Comment by mfusser** [comment]
I agree that might be useful for comparison. I want to have a look at it at some point anyway. 
I will add it to our backlog but I am not sure if/when I will have the time.

**Comment by itsezc** [comment]
@mfusser is it possible to add a changelog for each release

**Comment by Yogu** [comment]
> @mfusser is it possible to add a changelog for each release

Yeah we've been very negligent of the changelog for a while. I added [release notes for 2.1.0 and 2.0.0](https://github.com/AEB-labs/cruddl/releases) and try to add them to older releases gradually.

**Comment by frankmayer** [comment]
I just wanted to ask if any of the features discussed above have any chance at getting in. (Maybe in the time since the original post, some things have changed?)

I am especially interested in:

- Subscriptions (https://github.com/AEB-labs/cruddl/issues/128)
- Live Queries
- Seeding (https://github.com/AEB-labs/cruddl/issues/89), but this could also be done in application level

Thank you

**Comment by Yogu** [comment]
Sorry, none of these features are currently planned for a specific time frame.
